lib/modules/manager/unity3d/readme.md

```diff
@@ -0,0 +1,10 @@
+Handles version numbers inside `ProjectSettings/ProjectVersion.txt` files used by the Unity game engine.
+
+`ProjectSettings/ProjectVersion.txt` always contains two version references in a yml-like syntax; one with and one without a hash, both of which are updated:
```

Comment:
```suggestion
`ProjectSettings/ProjectVersion.txt` always have two version references in a yml-like syntax; one with and one without a hash, both are updated:
```

Also, is it YML-like syntax or YAML-syntax? I lean towards YAML-syntax, but not sure. 🙂